client/pimExtP_service.go

```diff
@@ -0,0 +1,57 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+)
+
+func (sm *ServiceManager) CreateExternalProfile(l3_outside string, tenant string, description string, pimExtPAttr models.PIMExternalProfileAttributes) (*models.PIMExternalProfile, error) {
```

Comment:
Should this not be CreatePimExternalProfile?
```suggestion
func (sm *ServiceManager) CreatePimExternalProfile(l3_outside string, tenant string, description string, pimExtPAttr models.PIMExternalProfileAttributes) (*models.PIMExternalProfile, error) {
```

Reply:
same applies for all naming in this file

Reply:
Also check to use PIM or Pim, because model uses PIM but then the next client file refers to CreatePimIPv6InterfaceProfile.

Reply:
ok